compiler/rustc_resolve/src/lib.rs

```diff
@@ -2458,8 +2458,14 @@ impl<'a> Resolver<'a> {
                             (format!("use of undeclared crate or module `{}`", ident), None)
                         }
                     } else {
-                        let mut msg =
-                            format!("could not find `{}` in `{}`", ident, path[i - 1].ident);
+                        let parent = path[i - 1].ident.name;
+                        let parent = if parent == kw::PathRoot {
+                            "crate root".to_owned()
```

Comment:
Should it be `crate root` instead?

Reply:
Hmmm, the path root is normally referred to as the crate root [as you can see here](https://doc.rust-lang.org/reference/paths.html#path-qualifiers). `root crate` makes it sound like this refers to a crate itself but it does not, it's the root of the namespace hierarchy.